com.unity.render-pipelines.core/Editor/Lighting/ProbeVolume/ProbeGIBaking.cs

```diff
@@ -291,6 +289,9 @@ static void OnBakeStarted()
             // Get min/max
             CellCountInDirections(out minCellPosition, out maxCellPosition, m_BakingProfile.cellSizeInMeters);
 
+            foreach (var data in ProbeReferenceVolume.instance.perSceneDataList)
```

Comment:
Do we need the full list or only the sceneDataList?

Reply:
since we have to force no blending while baking, we have to reset the full list, otherwise cells from other assets might still be blending states